src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -453,6 +453,13 @@ doclet.usage.linkoffline.parameters=\
 doclet.usage.linkoffline.description=\
     Link to docs at <url1> using package list at <url2>
 
+doclet.usage.link-modularity-mismatch.parameters=\
+    (warn|info)
```

Comment:
This is one of those cases where the words "warn" and "info" should *not* be localized, because they are the values accepted by the option -- assuming that those values are not themselves also localized.  I suggest you either add an immediately preceding comment, per the localization guidelines, or else use `({0}|{1})` and provide those strings from the calcite,